### PR TITLE
CFE-650: ALBO supports k8s service resource types using NLB

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/CFE-650

Link to docs preview: https://52309--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.html#nw-aws-load-balancer-operator_aws-load-balancer-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-aws-load-balancer-operator.adoc [hunk 1]
```diff
@@ -11,6 +11,8 @@ The AWS Load Balancer Operator can tag the public subnets if the `kubernetes.io/
 
 * Public and private subnets of the discovered VPC.
 
+The AWS Load Balancer Operator supports the Kubernetes service resource type `LoadBalancer` by using Network Load Balancer (NLB) with the `instance` or the `ip` target type.
```
Comment (thejasn) [inline modules/nw-aws-load-balancer-operator.adoc:14]:
```suggestion
The AWS Load Balancer Operator supports the Kubernetes service resource type `LoadBalancer` by using Network Load Balancer (NLB) with the `instance` target type only.
```

Comment (xenolinux) [reply modules/nw-aws-load-balancer-operator.adoc]: Addressed

**Comment by xenolinux** [comment]
@lihongan Would you please review this PR?

**Review by sheriff-rh — APPROVED** [review]
Looks good!

**Comment by lihongan** [comment]
Looks good. Thank you @xenolinux

File: modules/nw-aws-load-balancer-operator.adoc [hunk 2]
```diff
@@ -11,6 +11,8 @@ The AWS Load Balancer Operator can tag the public subnets if the `kubernetes.io/
 
 * Public and private subnets of the discovered VPC.
 
+The AWS Load Balancer Operator supports the Kubernetes service resource type `LoadBalancer` by using Network Load Balancer (NLB) with the `instance` target type only.
```
Comment (alebedev87) [inline modules/nw-aws-load-balancer-operator.adoc:14]:
```suggestion
The AWS Load Balancer Operator supports the Kubernetes service resource of type `LoadBalancer` by using Network Load Balancer (NLB) with the `instance` target type only.
```

Comment (xenolinux) [reply modules/nw-aws-load-balancer-operator.adoc]: Addressed

**Pull request merged**